ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -57,3 +57,52 @@ def test_log_poisson_loss(
         compute_full_loss=compute_full_loss,
         atol_=1e-2,
     )
+
+
+# hinge_embedding_loss
+@handle_test(
+    fn_tree="functional.ivy.experimental.hinge_embedding_loss",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=-3,
+        max_value=3,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=3,
+        min_dim_size=3,
```

Comment:
remove these unnecessary constraints, the test should be as unconstrained as possible
```suggestion

```